importer/BDCS/RPM/Files.hs

```diff
@@ -41,8 +41,8 @@ mkFiles rpm checksums =
         -- Is it worth catching that error here and doing... something?
         ty <- fromJust <$> getFileType mode
         let cksum = case lookup (T.pack path) checksums of
-                        Just (Just c) -> c
-                        _             -> "UNKNOWN"
+                        Just c -> c
+                        _      -> "UNKNOWN"
```

Comment:
With this change, I think you can replace the whole case with:

let cksum = fromMaybe "UNKNOWN" (lookup (T.pack path) checksums)